index.html

```diff
@@ -808,6 +811,53 @@ <h3>
           </p>
         </section>
       </section>
+      <section>
+        <h3>
+          `id` member
+        </h3>
+        <p>
+          The [=manifest's=] <code><dfn data-export="" data-dfn-for=
+          "manifest">id</dfn></code> member is a <a>string</a> that
+          represents the <dfn>id</dfn> for the application.
+        </p>
+        <p>The |id| is used  by user agents uniquely identify the application
+          globally. When |id| is changed, it's recognized as a new app and
```

Comment:
Globally might be confusion (as it might be confused with "global scope"), "universally" maybe? Assuming there cannot be collisions because another app in the same origin will effectively claim the "id".